dnf-behave-tests/features/steps/repo.py

```diff
@@ -298,9 +298,12 @@ def step_check_http_log(context, quantifier, command):
 
     if quantifier == 'every':
         assert all(matches), 'Every request should match the table: ' + dump
-    elif quantifier == 'exactly one':
-        assert len([m for m in matches if m]) == 1, \
-            'Exactly one request should match the table: ' + dump
+    elif quantifier.startswith('exactly '):
+        num = quantifier.split(' ')[1]
+        if num == 'one':
+            num = 1
+        assert len([m for m in matches if m]) == int(num), \
+            'Exactly %s requests should match the table: %s' % (quantifier, dump)
```

Comment:
In general I think it is preferable to state what is the error in the message, not what it should have looked like?

Reply:
Good point, I'll fix that.  For some reason I assumed that "documenting" what the `assert` is supposed to test is better, but it really turns out to be too confusing.  I'll fix that.

Reply:
Fixed in commit 6e4b108